src/components/core/PivotTable.tsx

```diff
@@ -285,13 +298,34 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
 
             const dataSourceChanged =
                 this.props.dataSource.getFingerprint() !== prevProps.dataSource.getFingerprint();
+            // TODO INE Should we support change of defaultWidth during lifetime??
```

Comment:
Please attempt to resolve the TODO (create follow-up or remove note if we decide not to support it)

Reply:
will add it later, when number will be possible value of "defaultWidth". Removed comment